modules/persistent-storage-csi-gcp-hyperdisk-limitations.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-gcp-hyperdisk-limitations_{context}"]
+= C3 and N4 instance type limitations
+The GCP PD CSI driver support for the C3 instance type for bare metal and N4 machine series have the following limitations:
+
+* Hyperdisk-balanced disks with RWX is not supported.
+
+* Storage pool cloning is not supported.
```

Comment:
Cloning volumes is not supported when using storage pools

Reply:
Good catch!